cpp/src/hash/concurrent_unordered_multimap.cuh

```diff
@@ -395,6 +398,8 @@ public:
                typename comparison_type = key_equal>
     __forceinline__
     __device__ iterator insert(const value_type& x,
+                               const int pass = 0,
```

Comment:
Should partitioned insert be a separate function? Or maybe a have a set_partition() function separate from insert and find? 

Or perhaps partitioned hash table should be a subclass? I think the best interface might be something different than you have here.

Reply:
Agreed. I think there should be a separate function for a "multi-pass" insert that just calls the regular `insert` function.

Reply:
1291316 implements separate methods `insert_part` and `find_part` that call the existing `insert` and `find` APIs (with pre-computed hash)